astropy/units/equivalencies.py

```diff
@@ -267,8 +279,24 @@ def iconverter_phot_f_nu_to_f_la(x):
         (phot_L_la, phot_L_nu, converter_phot_L_la_phot_L_nu, iconverter_phot_L_la_phot_L_nu),
         (phot_L_nu, L_nu, converter_phot_L_nu_to_L_nu, iconverter_phot_L_nu_to_L_nu),
         (phot_L_nu, L_la, converter_phot_L_nu_to_L_la, iconverter_phot_L_nu_to_L_la),
-        # surface brightness
+        # surface brightness (flux equiv)
         (S_la, S_nu, converter, iconverter),
+        (S_nu, nu_S_nu, converter_f_nu_to_nu_f_nu, iconverter_f_nu_to_nu_f_nu),
+        (S_la, la_S_la, converter_f_la_to_la_f_la, iconverter_f_la_to_la_f_la),
+        (phot_S_la, S_la, converter_phot_f_la_to_f_la, iconverter_phot_f_la_to_f_la),
+        (phot_S_la, S_nu, converter_phot_f_la_to_f_nu, iconverter_phot_f_la_to_f_nu),
+        (phot_S_la, phot_S_nu, converter_phot_f_la_phot_f_nu, iconverter_phot_f_la_phot_f_nu),
+        (phot_S_nu, S_nu, converter_phot_f_nu_to_f_nu, iconverter_phot_f_nu_to_f_nu),
+        (phot_S_nu, S_la, converter_phot_f_nu_to_f_la, iconverter_phot_f_nu_to_f_la),
+        # surface brightness (luminosity equiv)
```

Comment:
Are all the new converters tested?

Reply:
No.  As they are the same converters as for the flux density units, they are tested there.  Does not seem like they need to be tested a 2nd time as the only change is to add a per steradian units to both sides of the test.

Reply:
I'll defer to @mhvk .